doc/source/data/loading-data.rst

```diff
@@ -618,7 +542,7 @@ Call :func:`~ray.data.read_sql` to read data from a database that provides a
 
             pip install mysql-connector-python
 
-        Then, define your connection login and query the database.
+        Then, define your connection logic and query the database.
```

Comment:
IIUC, the following example will create a new connection for each `ray.data.read_sql`. This is an anti-pattern in practice. Do we want to update it to re-use one connection?

Reply:
It's necessary to create a new connection because connections aren't thread or process safe. So, we can't share a connection across read tasks.

Reply:
Those read_sql calls are sequential. So multi-threading isn't a problem?

Reply:
The `read_sql` calls are sequential, but `read_sql` creates connections in read tasks.